### DISCUSSION TITLE
Contributors willing to answer Issues and review PRs

### DISCUSSION THREAD

**Discussion by davidzollo** [apache/dolphinscheduler#7545] [Show and tell]
hi developers,

Thanks to the community.
Up to now, as the discussion in the **[dev mailing list](https://lists.apache.org/thread/9flwlzrp69xjn6v8tdkbytq8glqp2k51)**, we counted that the following contributors are willing to review PR and solve issues.
If more contributors are willing to join, the open source project is a long-term journey.  If you are interested in this topic, please feel free to leave a message

1, Master
@SbloodyS, @Linkaiup, @SunnyZ-L, @geosmart, @boy-xiaozhang (PRs), @yimaixinchen (PRs), @lvshaokang, @guws (PRs), @taotao (PRs)

2, Worker
@SbloodyS, @SunnyZ-L, @geosmart, @boy-xiaozhang (PRs), @yimaixinchen (PRs), @lvshaokang, @YangCH0915,  @Huzk8 (PRs)

3, K8S
@Linkaiup, @geosmart, @chengshiwen, @coding-now, @caoyj1991, @crazycarry

4, API
@Dev-mao (PRs), @Devosend (PRs), @Calvinjiang

5, Front-end
@Devosend (PRs), @Calvinjiang

6, Documentation
@lukairui, @dailidong, @Dev-mao (PRs), @aqi915,  @YangCH0915, @iJiangs, @Zhabei, @pengge00, @dukeshiny, @tracehh

7, Testing 
@aqi915, @jon-qj

**Comment by zhongjiajie** [comment]
FYI, I just add a guide about how to review issues and PRs for DolphinScheduler, if you do not know the detail, maybe you could take a look https://dolphinscheduler.apache.org/en-us/community/join/review.html

**Reply by davidzollo** [reply]
> FYI, I just add a guide about how to review issues and PRs for DolphinScheduler, if you do not know the detail, maybe you could take a look https://dolphinscheduler.apache.org/en-us/community/join/review.html

great job

**Comment by zhuxt2015** [comment]
Please add me to master and worker  ,about issues and review PR
